nameko_amqp_retry/backoff.py

```diff
@@ -124,44 +122,33 @@ def republish(self, backoff_exc, message, target_queue):
         expiration = backoff_exc.next(message, self.exchange.name)
         queue = self.make_queue(expiration)
 
-        # republish to appropriate backoff queue
+        properties = message.properties.copy()
+        headers = properties.pop('application_headers')
```

Comment:
I had en issue where in some cases (sorry, can't remember what) that property was not set.  Providing a default value (empty dict) would fix it.

Reply:
Thank you. I will fix this 👍